code/obj/item/material.dm

```diff
@@ -729,6 +741,11 @@
 	default_material = "telecrystal"
 	icon_state = "martian-bar"
 
+/obj/item/material_piece/telecrystal_ref
+	desc = "A cut block of Refined Telecrystal."
+	default_material = "telecrystal_ref"
+	icon_state = "martian-bar"
```

Comment:
👀

Reply:
shouldn't be a material at all imo. Just an item.

Reply:
them being a material allows for easy stacking, having slightly better than telecrystal matsci properties, and lets you make a traitor cry as they watch all their TC get put in a reclaimer

Reply:
More concerned mat duping making them unlimited